openbb_sdk/providers/polygon/openbb_polygon/models/stock_news.py

```diff
@@ -58,12 +57,11 @@ class PolygonPublisher(BaseModel):
 class PolygonStockNewsData(StockNewsData):
     """Source: https://polygon.io/docs/stocks/get_v2_reference_news"""
 
-    class Config:
-        fields = {
-            "url": "article_url",
-            "text": "description",
-            "date": "published_utc",
-        }
+    __alias_dict__ = {
+        "url": "article_url",
+        "text": "description",
+        "date": "published_utc",
+    }
 
     amp_url: Optional[str] = Field(description="AMP URL.")
```

Comment:
```suggestion
    amp_url: Optional[str] = Field(default=None, description="AMP URL.")
```

Apparently v2 requires us to explicitly set the default as `None` when type is `Optional[T]`
https://docs.pydantic.dev/2.0/migration/#required-optional-and-nullable-fields

Reply:
Yes, apparently `Optional` without a default value is a required field. Setting the default as `None` indicates it's not required.